plugins/inputs/aerospike/aerospike.go

```diff
@@ -21,6 +21,7 @@ import (
 //go:embed sample.conf
 var sampleConfig string
 
+// Aerospike Input Plugin
```

Comment:
Same here (and potentially everywhere else ;-)) What information does this comment add?

Reply:
Changed